### PR TITLE
user-crud: Implement Material IO in vue assignment.

### PR DESCRIPTION
**Pull request by nikhilatcompro**

### CODE REVIEW

File: vuejs/user-display-app/src/components/List.vue [hunk 1]
```diff
@@ -50,9 +55,9 @@
        * the value of showMore variable.
        * @param {user} user - the user for which we clicked the button.
        */
-      onShowMoreClick (user) {
-        // this.user = user;
-        this.$router.push(`/users/${user.id}`);
+      onShowMoreClick (userID) {
+        this.userID = userID;
+        // this.$router.push(`/users/${userID}`);
```
Comment (PurnimaCompro) [inline vuejs/user-display-app/src/components/List.vue:60]: Remove commented code

File: vuejs/user-display-app/src/components/List.vue [hunk 2]
```diff
@@ -62,69 +67,51 @@
...
-  }
-  .user-table > tr:nth-child(even){
-    background-color: white;
-  }
+  // .user-table > tr:nth-child(odd){
+  //   background-color: skyblue;
+  // }
+  // .user-table > tr:nth-child(even){
+  //   background-color: white;
+  // }
```
Comment (PurnimaCompro) [inline vuejs/user-display-app/src/components/List.vue:116]: Remove commented code

File: vuejs/user-display-app/src/components/MatListItem.vue [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    },
+    props: {
+      user: Object,
+      personIcon: String,
+      moreIcon: String
+    },
+    methods: {
+      onShowMoreClick () {
+          this.showMoreClicked = !this.showMoreClicked;
+          // this.$emit('clicked-more', userID);
```
Comment (PurnimaCompro) [inline vuejs/user-display-app/src/components/MatListItem.vue:39]: Remove commented code

File: vuejs/user-display-app/src/components/MatMenu.vue [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+      this.menu.open = false;
+    }
+  };
+</script>
+
+<style lang="scss" scoped>
+  .action-menu {
+    transform-origin: top right;
+    top: 0px;
+    right: 0px;
```
Comment (PurnimaCompro) [inline vuejs/user-display-app/src/components/MatMenu.vue:71]: when we specify 0, no need to mention rem>

just write top: 0

File: vuejs/user-display-app/src/components/List.vue (hunk 1 above)
Comment (nikhilatcompro) [reply vuejs/user-display-app/src/components/List.vue:60]: done

File: vuejs/user-display-app/src/components/List.vue (hunk 2 above)
Comment (nikhilatcompro) [reply vuejs/user-display-app/src/components/List.vue:116]: done

File: vuejs/user-display-app/src/components/MatListItem.vue (hunk 3 above)
Comment (nikhilatcompro) [reply vuejs/user-display-app/src/components/MatListItem.vue:39]: done

File: vuejs/user-display-app/src/components/MatMenu.vue (hunk 4 above)
Comment (nikhilatcompro) [reply vuejs/user-display-app/src/components/MatMenu.vue:71]: done

**Pull request merged**